### DISCUSSION TITLE
ManagedMediaSource

### DISCUSSION THREAD

**Discussion by kaambiz** [vidstack/player#933] [Q&A] [answered]
iOS Safari has been lacking support for MSE since forever (Apple claiming it's due to power usage inefficiency). For Safari 17 (on all their platforms) they opted for a new API called ManagedMediaSource. It's been proposed to [W3C](https://github.com/w3c/media-source/issues/320) but more importantly hls.js [merged](https://github.com/video-dev/hls.js/pull/5542) it into their next version.
Now, my question is will it be supported by Vidstack when hls.js 1.5.0 goes stable?

**Comment by mihar-22** [comment, marked as answer]
Yes definitely. Let me get this next release out this week, and then we can look into it.

**Reply by kaambiz** [reply]
Perfect as always. Thank you.
